src/style/reset.css

```diff
@@ -0,0 +1,8 @@
+.normalize,
```

Comment:
i d use reset.css for this

Reply:
I'm not sure I understand what i need to change here, since it is already in reset.css. Can you clarify, please?

Reply:
when resetting css you can use selectors by tags or even use a pre-written reset.css file https://meyerweb.com/eric/tools/css/reset/

Reply:
Got it, thanks for clarification!